test/functional/test-runtime.js

```diff
@@ -520,31 +517,44 @@ describe('runtime', () => {
       expect(getServiceForDoc(ampdoc, 'service1')).to.be.instanceOf(Service1);
     });
   });
+});
+
+
+describes.realWin('runtime multidoc', {
+  amp: {ampdoc: 'multi'},
+}, env => {
+  let win;
+  let extensions;
+  let extensionsMock;
+  let ampdocServiceMock;
+
+  beforeEach(() => {
+    win = env.win;
+    win.setTimeout = window.setTimeout;
```

Comment:
Is this because you want it to use the fake clock installed in the parent environment? That'll cause problems eventually.